### PR TITLE
Cleanup MANIFEST.in a bit

### PR DESCRIPTION
**Pull request by nicoddemus**
Remove some invalid entries and exclude files which should not go into a sdist

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -9,8 +9,6 @@ include HOWTORELEASE.rst
 include tox.ini
 include setup.py
 
-include .coveragerc
-
 include plugin-test.sh
 include requirements-docs.txt
```
Comment (nicoddemus) [inline MANIFEST.in]: Also, I wonder if we should remove those scripts as well? They all seem to be part of a release/testing workflow which is no longer true:

* `plugin-test.sh`: test a bunch of plugins located in parent directories using current pytest;
* `requirements-docs.txt`: we should advertise using `tox -e docs` instead.
* `runtox.py`: runs `tox` while pointing to @hpk42 index... it probably made sense when `py` and `pytest` release were really tied together, but now I don't think it makes sense.

Comment (The-Compiler) [inline MANIFEST.in]: All I can say is that I never used them :wink:

File: MANIFEST.in [hunk 2]
```diff
@@ -21,12 +19,12 @@ recursive-include extra *.py
...
-exclude _pytest/impl
-
 graft _pytest/vendored_packages
 
 recursive-exclude * *.pyc *.pyo
+recursive-exclude testing/.hypothesis *
+recursive-exclude testing/freeze/~ *
+recursive-exclude testing/freeze/build *
+recursive-exclude testing/freeze/dist *
+
```
Comment (The-Compiler) [inline MANIFEST.in]: Isn't this the same as doing `graft testing/.hypothesis` etc.?

**Comment by The-Compiler** [comment]
I think it's a good idea to define what should happen for *every* file, which is what [`check-manifest`](https://pypi.python.org/pypi/check-manifest) enforces.

Without your PR:

```
lists of files in version control and sdist do not match!
missing from sdist:
  .github
  .github/ISSUE_TEMPLATE.md
  .github/PULL_REQUEST_TEMPLATE.md
```

With your PR:

```
lists of files in version control and sdist do not match!
missing from sdist:
  .coveragerc
  .github
  .github/ISSUE_TEMPLATE.md
  .github/PULL_REQUEST_TEMPLATE.md
  _pytest/impl
  appveyor.yml
suggested MANIFEST.in rules:
  include *.yml
  include .coveragerc
```

So I think it's a bad idea to remove those lines, and we should probably `prune .github` too.

**Comment by nicoddemus** [comment]
I agree. We probably should make `check-manifest` part of `linting`, what do you think?

**Comment by The-Compiler** [comment]
I agree! And probably [pyroma](https://bitbucket.org/regebro/pyroma) too while we're at it.

Note that setuptools-scm breaks check-manifest though (or check-manifest breaks with setuptools-scm, whatever phrasing you prefer). We're not using it yet though.

**Comment by nicoddemus** [comment]
@RonnyPfannschmidt any inputs here?

I'm leaning towards declining this PR and opening a proper task for it, since this is growing out of the (small) scope I originally intended. :grin:

**Comment by nicoddemus** [comment]
Created #1907, thanks for the feedback!

**Pull request closed without merging**